docs/kubernetes/how-to-get-a-cluster-running.md

```diff
@@ -42,6 +56,7 @@ setenforce 0
 Disable `swap` because Kubernetes works with memory used onto server, which cannot be controlled if swap is turned on:
 ```bash
 swapoff -a
+sed -i '/ swap / s/^\(.*\)$/#\1/g' /etc/fstab
```

Comment:
source: https://askubuntu.com/a/465200

Reply:
It would be nice to add the source :+1:

Reply:
@vitek-rostislav I do not see resource for any other code, why should it be added here?

Reply:
@TomasLudvik Because this is the only one where I know the source. I thought that all the other commands are written here "from scratch", and this one is actually copypasted from the source mentioned. If this is true, the source should be mentioned IMHO.